docs/gpu-acceleration.md

```diff
@@ -132,6 +132,16 @@ pip install --force-reinstall llama_cpp_python==0.2.79 -C cmake.args="-DLLAMA_CU
 pip install instructlab/.
 ```
 
+If you are running Fedora 40, you need to replace the `Recompile llama-cpp-python using CUDA` section above with the
+following until [CUDA](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#host-compiler-support-policy)
+supports GCC v14.1+.
+
+```shell
+# Recompile llama-cpp-python using CUDA
+sudo dnf install clang17
+CUDAHOSTCXX=$(which clang++-17) pip install --force-reinstall llama_cpp_python==0.2.79 -C cmake.args="-DLLAMA_CUDA=on"
```

Comment:
```suggestion
CUDAHOSTCXX=$(command -v clang++-17) pip install --force-reinstall llama_cpp_python==0.2.79 -C cmake.args="-DLLAMA_CUDA=on"
```

which is not posix.

Reply:
Do we need a full path here?

Reply:
while technically true, i don't think this warrants blocking the PR -- `which` is available everywhere that these instructions are relevant to